README.md

```
@@ -1,10 +1,69 @@
+(Replace **1.3.0** with the appropriate released version.)
+
+For detailed usage examples and integration steps, refer to the **ID-Authentication repository**.
+
+## Documentation
+
+Additional documentation and design references are available in the main MOSIP documentation portal:  
+ https://github.com/mosip/documentation/tree/1.2.0/docs
+
+## Contribution & Community
+
+• To learn how you can contribute code to this application, [click here](https://docs.mosip.io/community/code-contributions).
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)